CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Repository Maintainers
+* @launchdarkly/team-sdk-go
```

Comment:
Looks like this repo isn't managed by terraform yet, thought it was.

Reply:
We can probably ignore the terraform thing. This really needs to be moved into the Go server SDK repo (can do later).